google/cloud/bigtable/table_bulk_apply_test.cc

```diff
@@ -223,6 +223,8 @@ TEST_F(TableBulkApplyTest, TooManyFailures) {
       bt::SingleRowMutation("foo", {bt::SetCell("fam", "col", 0_ms, "baz")}),
       bt::SingleRowMutation("bar", {bt::SetCell("fam", "col", 0_ms, "qux")})));
   EXPECT_FALSE(failures.empty());
+  EXPECT_EQ(google::cloud::v0::StatusCode::kAborted,
```

Comment:
Why `google::cloud::v0::StatusCode`? Can't this just be `google::cloud::StatusCode`?

Reply:
sure